lesson-17/src/index.js

```diff
@@ -0,0 +1,98 @@
+async function getTodos() {
+  const response = await fetch('https://dummyjson.com/todos?limit=3')
+  const todos = await response.json()
+
+  return todos.todos
+}
+
+const toDoList = document.querySelector('.todo-list')
+
+function makeToDoHTML(toDoArray) {
+  toDoArray.forEach((toDoItem) => {
+    const toDoBlock = document.createElement('div')
+    if (toDoItem.completed === true) {
```

Comment:
```suggestion
    if (toDoItem.completed) {
```

не нужно явно делать сравнение с `true`